wordpress-mobile/WordPress-iOS
# Moves User Analytics switch to App Settings

Closes #7431 

Moved the "Anonymous Usage Tracking" setting to App Settings (previously in Help & About). Renamed the label to "Send Statistics" and provided a footer with text describing what the heck the setting does. Also tweaked the label for the Aztec editor since I was there.

![simulator screen shot jun 24 2017 9 46 24 am](https://user-images.githubusercontent.com/373903/27509429-651c8f22-58c2-11e7-92f2-0d1c975e4eb8.png)

To test:
1. Build & launch the app.
2. Make sure Help & Support doesn't show the old switch.
3. Go into App Settings and find the switch. By default it should be on.
4. Turn off the switch, kill the app, relaunch, verify setting is still off.
5. Turn the setting back on, kill the app, relaunch, verify setting is on.

Needs review: @koke 

cc: @diegoreymendez (for updating the Aztec label)
